packages/browser/src/index.ts

```diff
@@ -53,4 +53,5 @@ export type { SpanStatusType } from '@sentry/core';
 export type { Span } from '@sentry/types';
 export { makeBrowserOfflineTransport } from './transports/offline';
 export { onProfilingStartRouteTransaction } from './profiling/hubextensions';
+export type { JSSelfProfiler } from './profiling/jsSelfProfiling';
```

Comment:
I assume this is not a type that is actually supposed to be consumed by our clients. If that is correct I would suggest not exporting the type here.

Reply:
Relic of the old approach where I exposed this on the carrier - I'm removing it.